src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamAllowTlsResumeTests.cs

```diff
@@ -13,6 +13,7 @@ namespace System.Net.Security.Tests
 {
     using Configuration = System.Net.Test.Common.Configuration;
 
+    [PlatformSpecific(TestPlatforms.Windows | TestPlatforms.Linux)]
```

Comment:
I think we should aslo add tests to verify that similar but not same sessions are _not_ resumed. So far we were rending on schannel so I think the test coverage is really weak. Since we implemented our own logic I think we need more tests.